app.js

```diff
@@ -22,4 +22,24 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then((beersResult) => {
+     res.render('beers.hbs', { beersResult });
+    })
+  .catch(error => console.log(error))
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+  .getRandom()
+  .then((responseFromAPI) => {
+     res.render('random-beer.hbs', { responseFromAPI });
+    })
+  .catch(error => console.log(error))
+});
+
+
+app.listen(3000, () => console.log('🏃running on port 3000'));
```

Comment:
It's considered good practice to have a callback to console.log which port your server is running on, I also recommend you get into the habit of storing your port number in a variable (it's commonly done as well)
```
const PORT = 3000;

app.listen(PORT, () => console.log(`Server is running on port ${PORT}`));